### DISCUSSION TITLE
Why do some of the releases have a .exe while others don't?

### DISCUSSION THREAD

**Discussion by FowadIjaz** [Serene-Arc/bulk-downloader-for-reddit#345] [Q&A] [answered]
Unsure if there's something I'm missing here

**Comment by Serene-Arc** [comment, marked as answer]
The releases up until v2.0.0 had an EXE file because they were compiled in a different way. Now, the BDFR is in a proper module format, so it can be called from the installed modules like anything else. The EXE is unnecessary and the current way is more cross-platform.

**Reply by yeeterwa** [reply]
Is there a guide on how to install it? I don't know python and stuff

**Reply by Serene-Arc** [reply]
Yes, there are instructions in the README. That's why it's titled "read me".

**Comment by aliparlakci** [comment]
Main advantage of releasing the project via pip instead of compiled binaries is to have the ability to install/update/uninstall the program from your system. Also, you can use bdfr from anywhere in your directory because pip add the location of the package to the PATH variable of your system.
